script/build-lib.ts

```diff
@@ -1,21 +1,18 @@
-import { mkdir, readFile, writeFile } from "node:fs/promises";
+import { readFile, writeFile } from "node:fs/promises";
 import { join } from "node:path";
 import { $ } from "bun";
-import { FontAssetType, OtherAssetType, generateFonts } from "fantasticon";
 
 const LIB_OUTPUT_DIR = "dist/lib/@cubing/icons";
 const OUTPUT_ICONS_CSS = join(LIB_OUTPUT_DIR, "cubing-icons.css");
 
-await mkdir(LIB_OUTPUT_DIR, { recursive: true });
-
-await generateFonts({
-  inputDir: "src/svg/",
-  outputDir: LIB_OUTPUT_DIR,
-  fontTypes: [FontAssetType.WOFF2],
-  assetTypes: [OtherAssetType.TS, OtherAssetType.CSS],
-  selector: ".cubing-icon",
-  name: "cubing-icons",
-});
+// There's an issue with using `ttf2woff2` with `bun` in CI: https://github.com/cubing/icons/issues/153
+// So we invoke `node` as a fallback.
+try {
+  // We use shell invocation instead of `import(…)` so we can recover from the crash.
+  await $`bun run ./script/build-lib-fantasticon-workaround.js`;
+} catch {
+  await $`node ./script/build-lib-fantasticon-workaround.js`;
+}
```

Comment:
(Just leaving a comment here so I can link to it)

I'm hoping if we simplify our use of nix to just use it to get our hands on `bun`, and let bun go on to (impurely) do what it needs to do, we won't need this hack.

Reply:
I think that worked! Please see https://github.com/cubing/icons/pull/156